gufe/components/solventcomponent.py

```diff
@@ -50,7 +51,7 @@ def __init__(
         neutralize : bool, optional
           if the net charge on the chemical state is neutralized by the ions in
           this solvent component.  Default `True`
-        ion_concentration : openff-units.unit.Quantity, optional
+        ion_concentration : openff-units.units.Quantity, optional
```

Comment:
```suggestion
        ion_concentration : openff.units.unit.Quantity, optional
```

Reply:
This doesn't work - mypy complains when you do that.

Reply:
(the above comment is about importing quantity directly from OpenFF.units.units). Effectively as far as I can tell there's a problem with the way Quantity is being typed in that file (since it's not a direct import of just pint.Quantity) which throws a type checking when mypy compares magnitudes.

We could import the `_Quantity` from that file though, which would be effectively like importing Quantity from pint.